### DISCUSSION TITLE
[Question] What is `Type: HTTPS, Plain DNS`?

### DISCUSSION THREAD

**Discussion by robertying** [AdguardTeam/AdGuardHome#3109] [Q&A] [answered]
Hi AdguardTeam,

Thanks for the amazing work you guys put into AdGuardHome. 

I have seen lots of DNS queries with `Type: HTTPS, Plain DNS` when I don't even have DoH enabled.

Last time I checked, there is no actual HTTPS record for DNS. So what exactly does Adguard receive in such cases?

And since I do not enable DoH, is there a way for me to block this kind of "query"?

**Comment by ameshkov** [comment, marked as answer]
* Type: HTTPS - this is a type of the DNS query: https://datatracker.ietf.org/doc/draft-ietf-dnsop-svcb-https/00/?include_text=1
* Plain DNS - means that the query was received over plain DNS protocol.

**Reply by robertying** [reply]
Thanks for answering @ameshkov. I tried to find HTTPS query via Google but got no luck.

So for this kind of query type, can I assume that some clients or apps are just doing something fancy and I don't have to care much?

**Reply by robertying** [reply]
Ah right after I posted the above, I found a blog post from Cloudflare stating Apple has started using HTTPS query to speed up things on iOS 14. https://blog.cloudflare.com/speeding-up-https-and-http-3-negotiation-with-dns/

Thanks for your time. I'll leave the link for anyone else who are curious about the recent HTTPS query spike.
